prime-router/quick-test.sh

```diff
@@ -277,7 +277,7 @@ fi
 if [ $RUN_OH -ne 0 ]
 then
   echo Generate fake OH data, HL7!
-  text=$(./prime data --input-fake 50 --input-schema oh/oh-covid-19 --output-dir $outputdir --target-states OH --target-counties Ashtabula --output-format HL7_BATCH)
+  text=$(./prime data --input-fake 50 --input-schema oh/oh-covid-19 --output-dir $outputdir --target-states OH --target-counties Ashtabula --output-format HL7_BATCH --suppress-qst-for-aoe)
```

Comment:
Added flag to test QST suppression

Reply:
I'll look for you in old Honolulu
San Francisco or **Ashtabula**
You're gonna have to leave me, now I know
But I'll see you in the sky above
In the tall grass and the ones I love
You're gonna make me lonesome when you go